orc8r/gateway/go/services/bootstrapper/gateway_info/get_info.go

```diff
@@ -44,7 +45,22 @@ func GetFormatted() (string, error) {
 			config.GetMagmadConfigs().BootstrapConfig.ChallengeKey, err)
 		return res, err
 	}
-	res = fmt.Sprintf("%s\nChallenge Key:\n--------------\n%s\n",
-		res, base64.StdEncoding.EncodeToString(marshaledPubKey))
+	res = fmt.Sprintf("%s\nChallenge Key:\n--------------\n%s\n%s\n",
+		res, base64.StdEncoding.EncodeToString(marshaledPubKey), PrintBuildInfo())
+
 	return res, nil
 }
+
+// PrintBuildInfo will return info about the build, if env vars exist available
+func PrintBuildInfo() string {
+	return fmt.Sprintf(
+		"Build Info\n" +
+		"\tCommit Branch: %s\n" +
+		"\tCommit Tag: %s\n" +
+		"\tCommit Hash: %s\n" +
+		"\tCommit Date: %s\n",
+		os.Getenv("MAGMA_BUILD_BRANCH"),
+		os.Getenv("MAGMA_BUILD_TAG"),
+		os.Getenv("MAGMA_BUILD_COMMIT_HASH"),
+		os.Getenv("MAGMA_BUILD_COMMIT_DATE"))
+}
```

Comment:
do you really need go code in every binary which reads & prints out 4 environment variables?

printenv | grep MAGMA_BUILD_ would do just that in a pinch, also - anyone can use show_gateway_info.py to do it